content/guides/cli/scripts/deprovision-users.md

```diff
@@ -86,6 +86,13 @@ For example:
      Managed User 1,ManagedUser1@test.com
    ```
 
+Set the user ID which will be the new owner of user content
+   before deleting the user, set the `NewFilesOwnerID` parameter to the
+   user ID you want.
+   If no value is provided before running the script, it will prompt you to
+   input one or press Enter to use the user ID of the current authenticated
+   user.
```

Comment:
```suggestion
   If no value is provided before running the script, it will prompt you to
   provide one or press **Enter** to use the user ID of the currently authenticated
   user.
```